<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>544</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>544</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>douban.fm</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="ChannelWidget" name="channelWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>56</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>544</width>
     <height>56</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>544</width>
     <height>56</height>
    </size>
   </property>
  </widget>
  <widget class="ControlPanel" name="controlWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>544</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>544</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="PauseMask" name="pauseWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="LyricWidget" name="lyricWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>544</width>
     <height>151</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>544</width>
     <height>151</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>544</width>
     <height>151</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LyricWidget</class>
   <extends>QWidget</extends>
   <header>lyricwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChannelWidget</class>
   <extends>QWidget</extends>
   <header>channelwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControlPanel</class>
   <extends>QWidget</extends>
   <header>controlpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PauseMask</class>
   <extends>QWidget</extends>
   <header>pausemask.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
